services/app-web/src/pages/StateSubmission/ReviewSubmit/ReviewSubmit.tsx

```diff
@@ -215,56 +215,58 @@ export const ReviewSubmit = ({
                 </ModalToggleButton>
             </PageActionsContainer>
 
-            <Modal
-                modalRef={modalRef}
-                id="review-and-submit"
-                modalHeading={
-                    unlocked ? 'Summarize changes' : 'Ready to submit?'
-                }
-                submitButtonProps={{ className: styles.submitButton }}
-                onSubmitText={unlocked ? 'Resubmit' : undefined}
-                onSubmit={submitHandler}
-                isSubmitting={formik.isSubmitting || submitMutationLoading}
-            >
-                {unlocked ? (
-                    <form>
+            {!isSessionExpiring && ( // if the session is expiring, close this modal so the countdown modal can appear
```

Comment:
🤔 is there anyway to add this into shared `Modal` component so we always give priority to our system alerts? Otherwise we have to remember to add this conditional to every Modal we add to the app. 

One idea- we get the `isSessionExpiring`  value from within the modal itself, and if  that value is true and we are not in the session priority modal (check by component `id`), we self close the non-priority modal.

Reply:
Good idea.  Done.